src/common/ColorThemeWorker.cpp

```diff
@@ -59,7 +59,7 @@ const QStringList ColorThemeWorker::radare2UnusedOptions = {
 
 ColorThemeWorker::ColorThemeWorker(QObject *parent) : QObject (parent)
 {
-    char* szThemes = r_str_home(R2_HOME_THEMES);
+    char* szThemes = r_xdg_datadir ("cons");
```

Comment:
These apis are only vailable on git versions of r2. Ill merge and add an if guard to make it safe for older r2